cmd/virt-chroot/tap-device-maker.go

```diff
@@ -20,6 +20,17 @@ func createTapDevice(name string, owner uint, group uint, queueNumber int) error
 		Group:      uint32(group),
 	}
 
+	// the tap device consumer (libvirt) needs to honor the multi-queue flag
+	// set by the creator. Libvirt will use the flag based on the number of
+	// vCPUs requested to it - when 1 vCPU is requested, libvirt will open the
```

Comment:
Sorry for bugging you again here, but libvirt does not create the vcpu correlation. We do that. We instruct libvirt to create only one queue if there is one vcpu. We can have more queues than vcpus or less.

Reply:
Omitted that. Hope it is better now.